### DISCUSSION TITLE
New device-types request

### DISCUSSION THREAD

**Discussion by HawkKaymer** [netbox-community/devicetype-library#2302] [Q&A] [answered]
Hi All,
I'm looking for the following device types to upload to Netbox:

Huawei
- CE9860
- CE8875
- CE8851
- CE6881
- NE8000-X8

**Comment by harryajc** [comment, marked as answer]
HI @HawkKaymer 

Please look over the [CONTRIBUTING.md](https://github.com/netbox-community/devicetype-library/blob/master/CONTRIBUTING.md) which gives a guide on how to raise a PR for these devices and then we will review it.

Harry
